### ISSUE TITLE
Intrested in contribution/maintaining

### ISSUE THREAD

**Issue by xobust** [ScottKaye/CoreRCON#17]
I forked and improved this library at work. 
[Challengermode/CoreRcon](https://github.com/Challengermode/CoreRcon)

I could help maintain and improve the project. 
We use it in production at work so it seems stable. 

Do you think we should merge or split it into a new repo since a changed how a connection is handled?

**Comment by ScottKaye** [comment]
Hi @xobust!

Your fork looks awesome!  I'd love to give away this project, especially to somebody who clearly maintains it well.  Thank you for reaching out!

I don't think having two repos would add any value for users.  If, for some reason, users want to stick with my old implementation with TCP listeners, they can install version 3.0.0.

I think the easiest way to transfer ownership would be to add you as a package owner to the NuGet package, then you could remove me.  I'll put up a notice in this project's README to point to your repo.  If you're interested, would you be able to tell me your NuGet username?

**Comment by xobust** [comment]
Hi @ScottKaye, 
I'm glad you liked my rewrite. 

I can do some more testing to check if the other listed games still work.
I'm not sure what is best, keeping the package name and warning the users to change the connection code or create a new package for the new versions. 

My NuGet username is [xobust](https://www.nuget.org/profiles/xobust).

**Comment by ScottKaye** [comment]
Testing if the other games still work would be useful, however since your fork still implements the same base API.  I originally wrote this for TF2, and I noticed you [removed support for that here](https://github.com/Challengermode/CoreRcon/commit/799d93c68e0a6b0918772be4d6693731afc36999#diff-04c6e90faac2675aa89e2176d2eec7d8).  It probably still works in all likelihood since both games are Valve/Source, but it would be nice to keep that going.

Publishing your package with version 4.0.0 would make the most sense to me.  Users upgrading a major version of a dependency should anticipate some code changes on their end.  As far as I can tell, it really just involves adding `await rcon.ConnectAsync();` after constructing an `RCON` object, which is pretty tame 👍 

Either way, I've added you as a package owner on NuGet.  I'm excited to see what you do with it!
